endpoints/users.py

```diff
@@ -33,4 +33,27 @@ def update_user_profile(id):
     user.rank = data.rank
     user.progression_data = data.progression_data
     db.session.commit()
-    return json.dumps(user.serialize(), cls=UUIDEncoder)
\ No newline at end of file
+    return json.dumps(user.serialize(), cls=ModelEncoder)
+
+
+@app.route("/user/<id>/split", methods=['GET'])
+def get_user_splits(id):
+    data = []
+    splits = Splits.query.filter_by(user_id=id).all()
+    if not splits:
+        if Users.query.filter_by(discord_id=id).first() is None:
+            return "Could not find User", 404
+    for row in splits:
+        data.append(row.serialize())
+    return data
+
+
+@app.route("/user/<id>/split/total/", methods=['GET'])
+def get_user_total_splits(id):
+    if Users.query.filter_by(discord_id=id).first() is None:
+        return "Could not find User", 404
+    data = 0
+    splits = Splits.query.filter_by(user_id=id).all()
+    for row in splits:
+        data += row.split_amount
```

Comment:
```suggestion
        data += row.split_contribution
```

AttributeError: 'Splits' object has no attribute 'split_amount'